### DISCUSSION TITLE
What does the "Accepted resourcepack" packet actually do? And can it be its own mod?

### DISCUSSION THREAD

**Discussion by Hlebuw3k** [ORelio/Minecraft-Console-Client#2028] [Q&A] [answered]
First of all i have almost no idea how coding works, so thats why im asking. Does this client just send the packet and ignore the resourcepack? If so, can this feature potentially theoretically be its own minecraft mod, to disable server enforced resource packs? Im asking this because of the 20w45a "require-resource-pack" server config, and im playing on a server with the servers resourcepack already installed locally, so i dont want to load the same resource pack again every time i join.

**Comment by ORelio** [comment, marked as answer]
Yes this is what we do in MCC, we just send back that we accepted and loaded the resource pack although we don't. This is because some servers have this as a requirement, but since MCC has no graphics, loading a resource pack does not make sense.

This could be theoretically implemented as a mod in Minecraft, but we don't mod Minecraft itself as part of the MCC project.
